backend/src/akvo/lumen/component/error_tracker.clj

```diff
@@ -4,12 +4,14 @@
             [raven-clj.core :as raven]
             [integrant.core :as ig]
             [raven-clj.ring]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?